docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -110,23 +101,23 @@ A tax provider is ready to establish a connection with the Tax Provider API when
 
-In either case, the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint should be called after the tax provider's app has been successfully installed. Tax providers will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
+In either case, the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint should be called after the tax provider's app has been successfully installed and the merchant's account credentials have been authenticated. Tax providers will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
 
 We recommend calling the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint immediately after the app has been successfully installed, otherwise your tax provider will not be displayed when merchants navigate to the **Settings > Setup > Tax** page in the control panel.
 
 The [Get a Connection](/docs/rest-contracts/tax-app-connection#get-connection-status) request may be used at any time to retrieve the connection status of the specified tax provider in the context of a store.
 
-## Enabling tax providers in the control panel
+## Configure tax provider settings in the control panel
 
-Once you have successfully installed the tax provider's app and provided basic authentication credentials through the Update a Connection request, merchants can enable the tax provider on all test stores supplied by the tax provider in [sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce).
+Once you have successfully installed the tax provider's app and provided basic authentication credentials through the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) request, merchants will be able to adjust the tax provider's settings to activate supported functions on their store. 
 
-To enable the tax provider, merchants must navigate to **Settings > Setup > Tax** in the control panel and click **Enable** next to the associated tax provider.
+The tax provider's settings screen can be found by navigating to **Settings > Setup > Tax > _TaxProviderName_** in the control panel. 
 
-If document submission is supported, navigate to the provider's settings menu, found at **Settings > Setup > Tax > _TaxProviderName_** in the control panel and ensure the **Submit Order Data** checkbox is checked.
+To enable the tax provider to respond to tax estimate requests, the merchant must choose which supported countries or subdivisions the tax provider will fulfill tax quotes for.
+
+If document submission is supported by the tax provider, the merchant can choose to select the **Submit Order Data** checkbox. This will enable document submission on their store, for supported transactions.
```

Comment:
In the above section we are introducing a significant change to the process in that we need the merchant to select supported countries or subdivisions in order for tax estimation to function.

e.g. A tax provider may support Australia and New Zealand, and the merchant may choose to select that only Australian based transactions are handled by the tax provider, for their purpose.

Reply:
👍